reference/library/src/morpheus-master/sass/modules/more-sites/_base.scss

```diff
@@ -63,6 +109,38 @@ ul#otherSitesMenu{
 	}
 }
 
+$narrowDisplay: "only screen and (max-width : 1200px)";
```

Comment:
Hi, great work indeed.

Could you add this breakpoint to _defaults.scss with `$break-small-end` and `$break-medium-end` to keep it organized.